services/blockchain-indexer/shared/dataService/business/token.js

```diff
@@ -68,6 +82,47 @@ const getTokenBalances = async (params) => {
 	return tokens;
 };
 
+const getTokenTopBalances = async (params) => {
+	const response = {
+		data: {},
+		meta: {},
+	};
+	const accountBalancesTable = await getAccountBalancesTable();
+	const queryParams = {
+		tokenID: params.tokenID,
+		sort: params.sort,
+		offset: params.offset,
+		limit: params.limit,
+	};
+
+	const tokenInfos = await accountBalancesTable.find(queryParams, ['address', 'balance']);
```

Comment:
Cant we directly pass params?
```suggestion
	const tokenInfos = await accountBalancesTable.find({...params}, ['address', 'balance']);
```